### DISCUSSION TITLE
Error al parchar imagen del sistema

### DISCUSSION THREAD

**Discussion by Darlin19902015** [bmax121/APatch#728] [Q&A]
Al momento de parchar me aparece un error no he podido solucionar, cabe indicar que ya tengo una versión anterior de Apatch esto sucede al actualizar 

https://github.com/user-attachments/assets/1215758c-ad13-43ee-9ac1-34cc1d9f15ab

**Comment by marat2509** [comment]
Duplication of https://github.com/bmax121/APatch/issues/725
